tests/test_coordinate_source.py

```diff
@@ -266,7 +267,7 @@ def createTestCtfFiles(self, index):
             "_rlnSphericalAberration": 700 + index,
             "_rlnAmplitudeContrast": 600 + index,
             "_rlnVoltage": 500 + index,
-            "_rlnDetectorPixelSize": 400 + index,
+            "_rlnMicrographPixelSize": 400 + index,
```

Comment:
Just curious: why this change?

Reply:
We had the wrong relion metadata column here, I believe. 

The output of our CTF estimator was changed by #817 

Discussion in the CTF validation issue:

https://github.com/ComputationalCryoEM/ASPIRE-Python/issues/742#issuecomment-1361697646

Reply:
Ok got it.